include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -1487,7 +1487,7 @@ __remove_elements(_ExecutionPolicy&& __exec, _ForwardIterator __first, _ForwardI
             ::std::forward<_ExecutionPolicy>(__exec), __result, __result + __m,
             [__result, __first, __is_vector](_Tp* __i, _Tp* __j) {
                 __invoke_if_else(
-                    ::std::is_trivial<_Tp>(),
+                    ::std::is_trivially_destructible<_Tp>(),
```

Comment:
Seems like, that the only place where we have this branch with `brick_move` and `brick_move_destroy`. Since we already modify the code around, let's drop this branch.

Reply:
Fixed